hasura/migrations/default/1657113614850_alter_table_public_circle_integrations_add_column_created_at/up.sql

```diff
@@ -0,0 +1,2 @@
+alter table "public"."circle_integrations" add column "created_at" timestamptz NOT NULL DEFAULT now()
+ null;
```

Comment:
you can remove the null here

Reply:
Removed with https://github.com/coordinape/coordinape/pull/1118/commits/4eddf3a51e1aa9a299b95f18b6ec701964ef7a4a, committed the GitHub suggestion and missed that